components/gardener/extensions/deployment.yaml

```diff
@@ -231,6 +231,38 @@ extension_manifests:
       - kind: BackupEntry
         type: aws
 
+########################################
+  provider-alicloud:
+########################################
+    <<: (( &template ))
+    apiVersion: core.gardener.cloud/v1beta1
+    kind: ControllerRegistration
+    metadata:
+      name: provider-alicloud
+    spec:
+      deployment:
+        providerConfig:
+          chart: (( encoded_chart ))
+          values:
+            image:
+              repo: (( version.image_repo || ~~ ))
+              tag: (( version.image_tag || ~~ ))
+            resources:
+              limits:
+                memory: 1Gi
+        type: helm
+      resources:
+        - kind: Infrastructure
+          type: alicloud
+        - kind: ControlPlane
+          type: alicloud
+        - kind: Worker
+          type: alicloud
+        - kind: BackupBucket
+          type: alicloud
+        - kind: BackupEntry
+          type: alicloud
+
```

Comment:
I think if you can run the garden cluster in mainland China, you should add image overwrite section just like below:
```yaml
imageVectorOverwrite: |
          images:
          - name: alicloud-controller-manager
            repository: registry.cn-shanghai.aliyuncs.com/gardener-accanary/gardener-de_alibaba-cloud-controller-manager
            sourceRepository: https://github.com/kubernetes/cloud-provider-alibaba-cloud
            tag: sha256:7c693eca4a1da22b199f8e8fda2fcaaf4ebf2dd9a2a5b9a7cfaa68ca7de4db18
         - ...
```

Reply:
good points.